arcgis-ios-sdk-samples/Scenes/Create terrain surface from a local tile package/CreateTerrainSurfaceFromLocalTilePackageViewController.swift

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+import UIKit
+import ArcGIS
+
+class CreateTerrainSurfaceFromLocalTilePackageViewController: UIViewController {
+    @IBOutlet weak var sceneView: AGSSceneView! {
+        didSet {
+            //initialize the map with imagery basemap
+            let scene = AGSScene(basemapType: .imageryWithLabels)
+            sceneView.scene = scene
+            
+            //set initial map area
+            let camera = AGSCamera(latitude: 36.525, longitude: -121.80, altitude: 300.0, heading: 180, pitch: 80, roll: 0)
+            sceneView.setViewpointCamera(camera)
+            
+            // load the tile package and add it as an elevation source
```

Comment:
"Load" is a separate concept in our API. Could you change it to  "Create an elevation source using the path to the tile package" or something like that?

and a separate comment for adding the elevation source to surface